README.md

```diff
@@ -3,10 +3,28 @@
 [![Travis CI Status](https://secure.travis-ci.org/openlayers/ol3.svg)](http://travis-ci.org/#!/openlayers/ol3)
 [![Coverage Status](https://coveralls.io/repos/openlayers/ol3/badge.svg?branch=master)](https://coveralls.io/r/openlayers/ol3?branch=master)
 
-Welcome to [OpenLayers 3](http://openlayers.org/)!
+[OpenLayers 3](http://openlayers.org/) is a high-performance, feature-packed library for creating interactive maps on the web.
 
-## Examples
-Check out the [hosted examples](http://openlayers.org/en/master/examples/), the [workshop](http://openlayers.org/ol3-workshop/) or poke around the evolving [API docs](http://openlayers.org/en/master/apidoc/).
+## Getting Started
+
+- Download the [latest release](https://github.com/openlayers/ol3/releases/tag/v3.4.0)
```

Comment:
The idea is that we will have a downloads page that will be updated with every release (e.g. http://openlayers.org/download/).  When that is published, we will update the readme, but the url won't change with every release.